.env.production.sample

```diff
@@ -11,6 +11,9 @@ DB_PORT=5432
 LOCAL_DOMAIN=example.com
 LOCAL_HTTPS=true
 
+# Use this if mastodon has to run on a different domain as the one used for federation
+#WEB_DOMAIN=mastodon.example.com
```

Comment:
I would add a note that people should leave it commented out unless they know exactly what they are doing - and add a space after `#` for consistency.